src/Components/ProductsPage.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import data from '../data.json';
+import ProductTable from './ProductTable';
+import SearchBar from './SearchBar';
+
+
+function ProductsPage () {
+const [products, setProducts] = useState(jsonData);
+const [productTable, setProductsTable] = useState(jsonData)
+const [SearchBar, setSearchBar] = useState()
+const [stock , SetStock]= useState(false)
+
+
+  return(
+      <div>
+        <h1>IronStore</h1>
+      </div>    
+      <div className="product-container">
+      {productTable.map((element) => (
+        <products
```

Comment:
when you map you return a component. Here it seems you are trying to return a state variable